capture-sdk/sdk/src/main/java/net/gini/android/capture/util/UploadMetadata.kt

```diff
@@ -0,0 +1,84 @@
+package net.gini.android.capture.util
```

Comment:
[Request] `UploadMetadata` is closely coupled to the documents and so I think it would fit better into the `net.gini.android.capture.document` package.